src/pkynetics/data_import/dilatometry_importer.py

```
@@ -26,43 +34,49 @@ def dilatometry_importer(file_path: str) -> Dict[str, np.ndarray]:
 
     try:
         # Detect file encoding
-        with open(file_path, 'rb') as file:
+        with open(file_path, "rb") as file:
             raw_data = file.read()
-            result = chardet.detect(raw_data)
-            encoding = result['encoding']
+            detection_result: DetectionResult = chardet.detect(raw_data)
```

Ruff (0.8.0)

Local variable `detection_result` is assigned to but never used

Remove assignment to unused variable `detection_result`

(F841)